src/scss/pages/_pricing.scss

```diff
@@ -0,0 +1,86 @@
+.Pricing {
+    &-plans {
+        align-items: center;
+        display: flex;
+        height: 878px;
+        justify-content: center;
+        width: 100%;
+    }
+
+    &-container {
+        width: 1110px;
+    }
+
+    &-title {
+        align-items: center;
+        color: $button-color;
+        display: flex;
+        font-family: "Rajdhani", sans-serif;
+        font-size: 30px;
+        font-weight: 100;
+        justify-content: center;
+        margin-bottom: 85px;
+        width: 1110px;
+    }
+
+    &-box {
+        display: flex;
+        justify-content: space-evenly;
+    }
+
+    &-card {
+        align-items: center;
+        background: linear-gradient(to right, $red-dark 0%, $red-color 51%, $red-dark 100%);
+        display: flex;
+        flex-direction: column;
+        height: 495px;
+        width: 348px;
+
+        &-title {
+            align-items: center;
+            color: $white-color;
+            display: flex;
+            font-size: 18px;
+            height: 80px;
+            justify-content: center;
+            margin: 0;
+            width: 100%;
+        }
+
+        &-price {
+            align-items: center;
+            background-color: $black-color;
+            color: $white-color;
+            display: flex;
+            display: flex;
+            font-weight: bold;
+            height: 100px;
+            justify-content: center;
+            justify-content: center;
+            margin: 0;
+            margin-bottom: 40px;
+            width: 100%;
+        }
+
+        &-list {
```

Comment:
Delete this and use the classes that bootstrap provides.

Reply:
done